updated contributing, ran script to add authors (added @su-vikas), fixed MD errors (#1533)
OWASP/owasp-mstg: CONTRIBUTING.md

- **Review pull requests for technical content**. Do you have tons of experience with the technical subject described in the PR? Chip in with constructive feedback.
- **Proofread and fix errors**. If you are studying the MSTG, write down any error, no matter how small, and submit them in an issue or fix them yourself through a PR.
- **Contribute to the MSTG Apps**. Extend and maintain our existing mobile applications, which are the [Crackmes](https://github.com/OWASP/owasp-mstg/tree/master/Crackmes "MSTG Crackmes") and the applications on our [Hacking playground](https://github.com/OWASP/MSTG-Hacking-Playground "MSTG Hacking Playground").

> - **Contribute to the Crackme cracking**. If you create a write-up on how to crack the Crackme's or the hacking playground: contact us and we can together create a link to it.

Reviewer: ```suggestion
- **Contribute to the Crackme Write-ups**. If you create a write-up on how to crack the Crackmes or the Hacking Playground, we can gladly include a link to it: simply submit a PR adding your link to the [Crackmes README file](https://github.com/OWASP/owasp-mstg/tree/master/Crackmes).
```

Although we greatly appreciate any and all contributions to the project, there are a few things that you should take into consideration:

- The MSTG should not be used as a platform for advertisement of commercial tools, companies or individuals. Write-ups should be written with free and open-source tools in mind and commercial tools are typically not accepted, unless as a reference in the security tools section.

> - The MSTG should not be used as a blogging platform: write-ups in the **write-up folder** should be "new" with differnet techniques to crack a crackme, if information is already in the guide: do not duplicate it with additional blogs within the repository.

Reviewer: I thought we were removing this because we put it as part of the contributing. So no need to include this line.
```suggestion
```

Although we greatly appreciate any and all contributions to the project, there are a few things that you should take into consideration:

- The MSTG should not be used as a platform for advertisement of commercial tools, companies or individuals. Write-ups should be written with free and open-source tools in mind and commercial tools are typically not accepted, unless as a reference in the security tools section.

> - The MSTG should not be used as a blogging platform: write-ups in the **write-up folder** should be "new" with different techniques to crack a crackme, if information is already in the guide: do not duplicate it with additional blogs within the repository. Instead, we recommend to publish it on Medium or another blogging platform.

Reviewer: we don't have this possibility anymore, no need for this line
```suggestion
```
